README.rst

```diff
@@ -272,24 +272,26 @@ CARTO Credential Management
 Typical usage
 ^^^^^^^^^^^^^
 
-The most common way to input credentials into cartoframes is through the `Context`, as below. Replace `{your_user_name}` with your CARTO username and `{your_api_key}` with your API key, which you can find at ``https://{your_user_name}.carto.com/your_apps``.
+The most common way to input credentials into cartoframes is through the `set_default_credentials` method, as below. Replace `{your_user_name}` with your CARTO username and `{your_api_key}` with your API key, which you can find at ``https://{your_user_name}.carto.com/your_apps``.
 
 .. code:: python
 
-    from cartoframes.auth import Context
-    con = Context(
-        base_url='https://{your_user_name}.carto.com',
+    from cartoframes.auth import set_default_credentials
```

Comment:
Should we mention here that the `api_key` is only required for private datasets? It makes possible to set the credentials by using only the username: `set_default_credentials('{your_user_name}')`. Just a suggestion, don't want to add too much noise in this section.

Reply:
:+1: Done!